src/status_im/ui/screens/signing/views.cljs

```diff
@@ -90,14 +92,81 @@
     [react/view {:padding 6}
      [react/text {:style {:color colors/blue}} (i18n/label :t/cancel)]]]])
 
-(views/defview password-view [{:keys [type error in-progress? enabled?]}]
+(views/defview keycard-pin-view []
```

Comment:
maybe move to separate keycard view ?

Reply:
@flexsurfer you mean to separate file? anyway it will re-written once PIN screen from basic onboarding will be merged, i'll take it from there.